src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -1894,7 +1894,7 @@ BOOST_AUTO_TEST_CASE(BatchNormTest)
          << 2.0467 << arma::endr
          << 1.9356 << arma::endr;
 
-  CheckMatrices(output, result, 1e-1);
+  // CheckMatrices(output, result, 1e-1);
```

Comment:
why did we remove this existing test? It should be fine with the linear layer I guess.

Reply:
Yes it does, I was testing different threshold for linear region so I commented it out. Will fix it in the next commit.